h2o-algos/src/main/java/hex/tree/ScoreBuildHistogram2.java

```diff
@@ -299,7 +296,7 @@ public ComputeHistoThread makeCopy() {
     }
 
     @Override
-    protected void map(int id){
+    protected void map(int UNUSED){
```

Comment:
This change looks weird. I rather keep it as `id` if there is no solution for removing this from the parent class.

Reply:
ok, sure - will remove